packages/valory/contracts/gnosis_safe/contract.py

```diff
@@ -450,9 +451,17 @@ def get_raw_safe_transaction(  # pylint: disable=too-many-arguments,too-many-loc
         ):
             tx_parameters.update(ledger_api.try_get_gas_pricing(old_price=old_price))
         # note, the next line makes an eth_estimateGas call!
+
         transaction_dict = w3_tx.buildTransaction(tx_parameters)
-        transaction_dict["gas"] = Wei(
-            max(transaction_dict["gas"] + 75000, configured_gas)
+        gas_estimate = (
+            ledger_api._try_get_gas_estimate(  # pylint: disable=protected-access
+                transaction_dict
+            )
+        )
+        transaction_dict["gas"] = (
+            Wei(max(gas_estimate + 75000, configured_gas))
+            if gas_estimate is not None
+            else configured_gas
```

Comment:
Should we add in the method description that the `base_gas` and `safe_tx_gas` will not be considered if the `estimated gas + 7500` is larger?